nav2_docking/README.md

```diff
@@ -151,6 +153,10 @@ There are two functions used during dock approach:
    for charging to start by calling `isCharging`.
    * This may be implemented using the `sensor_msgs/BatteryState` message to check the power status or for charging current.
    * Used for charging-typed plugins.
+   + * `startDetectionProcess`: Start any detection pipelines required for pose refinement.
```

Comment:
These bullets are needing 1 less set of indentations, currently they're nested under `isCharging`. Also, then the `two functions` should be 4. 

Personally I Think these can be removed, these are just describing the 'main' functions, there are more than just these and I don't think these count as 'main' functions.